src/util/kaldi-table-inl.h

```diff
@@ -298,10 +338,38 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
     }
     std::string line;
     if (getline(script_input_.Stream(), line)) {
-      SplitStringOnFirstSpace(line, &key_, &data_rxfilename_);
-      if (!key_.empty() && !data_rxfilename_.empty()) {
+      std::string data_rxfilename_next, rest;
+      // rest: a temp string which may contain the range specifier.
```

Comment:
this description of 'rest' is not very clear- you need to back up and describe what it is at the top level before getting into details.  Or just remove the comment.